docs/components/optimize/userguide/processes.md

```diff
@@ -0,0 +1,31 @@
+---
+id: processes
+title: Processes
+---
+
+
+## Overview
+
+The processes section give you an overview of all the processes and their status on a single page.
+It allows you to set a process owner to take responsibility of the process and time goals to track its performance.
+
+![Processes page](./img/processes.png)
+
+
+
+## Setting an owner: 
+Assign the responsibility for process health to have the right conversation with the right person to fix a process. Once you open up the set owner modal, you can select one of the users in your organization to be the main person responsible for the process.
```

Comment:
nit: just a matter of taste, I don't think we need to highlight a reason but rather just indicate the meaning of an owner? wdyt about this?
```suggestion
Assign the owner for process health to make transparent who is the responsible individual for it. Once you open up the set owner modal, you can select one of the users in your organization to be the main person responsible for the process.
```